css/user.css

```diff
@@ -0,0 +1,208 @@
+
```

Comment:
- Merge your css code into classes
- Much of css is not needed, just use the corresponding bootstrap classes. 
- With the right usage of bootstrap grid you won't need any css for moving elements (ie margins/paddings)
- Make sure you understand the difference between (top/bottom, margin and padding)